src/main/routes/helm/charts/get-chart-route.injectable.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { routeInjectionToken } from "../../../router/router.injectable";
+import type { Route } from "../../../router";
+import { helmService } from "../../../helm/helm-service";
+import { apiPrefix } from "../../../../common/vars";
+import type { RawHelmChart } from "../../../../common/k8s-api/endpoints/helm-charts.api";
+
+interface GetChartResponse {
+  readme: string;
+  versions: RawHelmChart[];
+}
+
+const getChartRouteInjectable = getInjectable({
+  id: "get-chart-route",
+
+  instantiate: (): Route<GetChartResponse> => ({
+    method: "get",
+    path: `${apiPrefix}/v2/charts/{repo}/{chart}`,
```

Comment:
I have (what I think) is a pretty cool idea of how to make sure that this string is "correctly typed" with regards to the interface that `Params` expects below but we can do that later.